packages/lib/services/KeymapService.ts

```diff
@@ -303,6 +303,23 @@ const defaultKeymapItems = {
 		{ accelerator: 'Cmd+P', command: 'gotoAnything' },
 		{ accelerator: 'Shift+Cmd+P', command: 'commandPalette' },
 		{ accelerator: 'F1', command: 'help' },
+		{ accelerator: 'Cmd+D', command: 'editor.deleteLine' },
+		{ accelerator: 'Cmd+Z', command: 'editor.undo' },
+		{ accelerator: 'Cmd+Y', command: 'editor.redo' },
+		{ accelerator: 'Cmd+Home', command: 'editor.goDocStart' },
+		{ accelerator: 'Cmd+End', command: 'editor.goDocEnd' },
+		{ accelerator: 'Cmd+Left', command: 'editor.goGroupLeft' },
+		{ accelerator: 'Cmd+Right', command: 'editor.goGroupRight' },
+		{ accelerator: 'Alt+Left', command: 'editor.goLineStart' },
+		{ accelerator: 'Alt+Right', command: 'editor.goLineEnd' },
```

Comment:
On macOS, the default for those 2 should be Ctrl A and E

```
	{ accelerator: 'Ctrl+A', command: 'editor.goLineStart' },
	{ accelerator: 'Ctrl+E', command: 'editor.goLineEnd' },
```

Reply:
Thanks, I've changed this now!